src/modules/systemlib/system_params.c

```diff
@@ -125,7 +125,7 @@ PARAM_DEFINE_INT32(SYS_MC_EST_GROUP, 2);
  * @value 338400 Normal Telemetry (38400 baud, 8N1)
  * @value 357600 Normal Telemetry (57600 baud, 8N1)
  * @value 3115200 Normal Telemetry (115200 baud, 8N1)
- * @value 419200 Iridium Telemetry (19200 baud, 8N1)
+ * @value 4115200 Iridium Telemetry (115200 baud, 8N1)
```

Comment:
Isn't the default baud 19200? Doesn't this have to be changed manually in the module?

Reply:
Yes the default one of the module is 19200 but the iridiumsbd driver assumes that the module uses 115200. I actually do not know why we changed it as it the people who initially decided it already left. I assume that it had a specific reason because it is some small extra work. I would also like to keep it that way because with this configuration we did all the tests and are sure that it works which might not be the case with the lower baud rate and I do not have the time now for extensive testing now.

Reply:
I would still propose having an additional option for 19200 baud so that other interested users have a starting point with a stock radio.

Reply:
If I have time in the future I can add a baud rate option for the driver and then also add a different value for the 19200 baud. But right now the 19200 baud will not work and if the parameter is there, then people will use it and will be confused why it does not work. Unfortunately right now I don't have the time to modify the driver and test it enough.

Reply:
Ok, thanks for the update.